onnxruntime/test/providers/cpu/tensor/split_op_test.cc

```diff
@@ -438,7 +438,9 @@ TEST(SplitOperatorTest, SplitAttributeSumTooSmall) {
   outputs.push_back({{1, 2}, {1.f, 2.f}});
   outputs.push_back({{2, 2}, {3.f, 4.f, 5.f, 6.f}});
 
-  RunTest<float>(axis, splits, input, outputs, false, true, false,true, "Cannot split using values in 'split' attribute");  //TensorRT parser: Assertion failed: axis != BATCH_DIM
+  RunTest<float>(axis, splits, input, outputs, false, true, false, true,
```

Comment:
ONNX took in a commit where such issues are caught at shape inference level and hence needs a minor test change - https://github.com/onnx/onnx/commit/6a570ff2fe9da4f48eb4fbb6c9ee337b8fdbedd3 
(Seems trivial enough to just include it in this PR)